serveradmin/serverdb/query_committer.py

```diff
@@ -359,6 +361,12 @@ def _log_changes(commit, changed, created_objects, deleted_objects):
 
 
 def _fetch_servers(object_ids):
+    """ Fetch servers with row lock.
```

Comment:
Other docstrings don't have a space before and dot after the title.

Reply:
fixed